addons/prometheus/prometheus.yaml

```diff
@@ -229,14 +229,12 @@ spec:
                 interval: 30s
                 scheme: http
         prometheusSpec:
-          image:
-            tag: v2.19.2
           thanos:
             version: v0.8.1
```

Comment:
TODO later .. but we should upgrade this old version of thanos. and looks like there are some new thanos capabilities in these newer prom charts that we should look into and utilize if relevant :)